docs/data/data-grid/events/CatalogOfEventsNoSnap.js

```diff
@@ -43,102 +55,87 @@ apiRef.current.subscribeEvent(
+    },
+  },
+  {
+    field: 'description',
+    headerName: 'Description',
+    flex: 1,
+    renderCell: ({ value, colDef }) => {
+      const width = colDef.computedWidth - 20;
+      return (
+        <div
+          style={{ minWidth: width, maxWidth: width, width, whiteSpace: 'normal' }}
+          dangerouslySetInnerHTML={{
+            __html: value,
+          }}
+        />
+      );
+    },
+  },
+];
 
-  return (
-    <React.Fragment>
-      <TableRow>
-        <TableCell style={{ borderBottom: 'unset' }}>
-          <IconButton
-            aria-label="expand row"
-            size="small"
-            onClick={() => setOpen(!open)}
-          >
-            {open ? <KeyboardArrowUpIcon /> : <KeyboardArrowDownIcon />}
-          </IconButton>
-        </TableCell>
-        <TableCell style={{ borderBottom: 'unset' }}>
-          <code>{event.name}</code>
-          {planBadge}
-        </TableCell>
-        <TableCell style={{ borderBottom: 'unset' }}>
-          <div
-            dangerouslySetInnerHTML={{
-              __html: event.description,
-            }}
-          />
-          {!!event.params && (
-            <div>
-              <Typography variant="subtitle2" component="span">
-                Params:{' '}
-              </Typography>
-              <code
-                dangerouslySetInnerHTML={{
-                  __html: escapeHTML(event.params),
-                }}
-              />
-            </div>
-          )}
-          {!!event.event && (
-            <div>
-              <Typography variant="subtitle2" component="span">
-                Event:{' '}
-              </Typography>
-              <code
-                dangerouslySetInnerHTML={{
-                  __html: escapeHTML(event.event),
-                }}
-              />
-            </div>
-          )}
-        </TableCell>
-      </TableRow>
-      <TableRow>
-        <TableCell style={{ paddingBottom: 0, paddingTop: 0 }} colSpan={3}>
-          <Collapse in={open} timeout="auto" unmountOnExit>
-            <Box sx={{ margin: 1 }}>
-              <HighlightedCode code={example} language="tsx" />
-            </Box>
-          </Collapse>
-        </TableCell>
-      </TableRow>
-    </React.Fragment>
-  );
-};
+const Toolbar = () => (
+  <GridToolbarContainer>
+    <GridToolbarQuickFilter />
+  </GridToolbarContainer>
+);
 
 export default function CatalogOfEventsNoSnap() {
   return (
-    <MarkdownElement>
-      <Table>
-        <TableHead>
-          <TableRow>
-            <TableCell />
-            <TableCell align="left">Name</TableCell>
-            <TableCell align="left">Description</TableCell>
-            <TableCell />
-          </TableRow>
-        </TableHead>
-        <TableBody>
-          {events.map((event) => (
-            <EventRow key={event.name} event={event} />
-          ))}
-        </TableBody>
-      </Table>
-    </MarkdownElement>
+    <Box sx={{ width: '100%', height: 800 }}>
+      <DataGridPro
```

Comment:
Good idea to use the data grid itself to showcase the documentation of the events. I found one issue tho - when you open expand an event and scroll past it then scroll top glitches.

https://user-images.githubusercontent.com/5858539/177980783-9d6e90ba-8b91-4b92-89ad-00fe172468b6.mov

.

Reply:
:laughing: if it even help us find bugs
The previous page had an equivalent of an autoHeight grid so if the scroll is buggy we can put autoHeight while we fix the bug.